src/test/auth.spec.js

```diff
@@ -204,4 +217,113 @@ describe('/api/v1/auth', () => {
       res.body.error.should.equal('Password must contain at least one letter, at least one number, and be atleast 8 digits long');
     });
   });
+
+  describe('Verify User email', () => {
+    beforeEach(async () => {
+      send = sinon.stub(sgMail, 'send').resolves({});
+    });
+
+    afterEach(async () => {
+      send.restore();
+    });
+
+    it('should sign up a new user', async () => {
+      const res = await chai.request(app)
+        .post(`${URL_PREFIX}/signup`)
+        .send({
+          first_name: 'qqqq',
+          last_name: 'qqqq',
+          email: 'tees@trtr.com',
+          password: '11111111ghghjh'
+        });
+      res.should.have.status(201);
+      res.body.should.have.property('data');
+      res.body.data.should.have.property('token');
+      const { token } = res.body.data;
```

Comment:
Move this block of code up before your assertion

`
const { token } = res.body.data;
 userToken = token;
`